api/src/main/java/org/openmrs/validator/ConceptSourceValidator.java

```diff
@@ -52,7 +52,8 @@ public void validate(Object obj, Errors errors) throws IllegalArgumentException
 			        + ConceptSource.class);
 		} else {
 			ValidationUtils.rejectIfEmptyOrWhitespace(errors, "name", "error.name");
-			ValidateUtil.validateFieldLengths(errors, obj.getClass(), "name", "hl7Code", "description", "retireReason");
+			ValidateUtil.validateFieldLengths(errors, obj.getClass(), "name", "hl7Code", "uniqueId", "description",
```

Comment:
Does this mean that we are no longer able to save any existing concept source that has not yet been assigned a value to this new field?